apps/src/blockly/addons/blockSvgUnused.js

```diff
@@ -56,7 +56,7 @@ export default class BlockSvgUnused extends BlockSvgFrame {
       'mousedown',
       this,
       function (e) {
-        if (Blockly.utils.isRightButton(e)) {
+        if (Blockly.browserEvents.isRightButton(e)) {
```

Comment:
CDO definition in utils: https://github.com/code-dot-org/blockly/blob/v4.0.14/core/utils/utils.js#L383
Google definition in browser_events: https://github.com/google/blockly/blob/399bd650a69f50843f2b46a9907a8dce826f6b99/core/browser_events.ts#L184-L191